spec/prawn/vera_pdf.rb

```diff
@@ -1,4 +1,4 @@
-require 'nokogiri'
```

Comment:
We have `spec/extensions` dir for things like this. It probably should be renamed to `helpers` but that's how it is. For historical reasons.

Please put this file in that dir.